config/settings.yml

```diff
@@ -5,6 +5,7 @@ purl_conn_timeout: 2
 purl_feedback_email: 'some-email-address@distribution-list-host.stanford.edu'
 stacks_url: 'https://stacks.stanford.edu'
 iiif_info_url: 'https://library.stanford.edu/iiif/viewers'
+metrics_api_url: 'https://sdr-metrics-api-prod.stanford.edu'
```

Comment:
Do we want to configure all of our instances to send the data to the prod metrics server?  It seems this even will send the data I generate by running it locally.

Reply:
Updated so that it's not provided by default, and if there's no value set, we don't track views.